projects/sandbox/src/app/progress-tracker/progress-tracker.component.html

```diff
@@ -195,95 +195,100 @@ <h2 class="title-id">{{ datasetId }}</h2>
-        <div *ngIf="warningDisplayedTier === DisplayedTier.CONTENT">
-          <a (click)="closeWarningView()" class="warning-animated warning-view-title"
-            >Content Tier Zero ({{ contentTier.total }})</a
-          >
-          <ng-content
-            *ngTemplateOutlet="
-              warningViewHeader;
-              context: { list: contentTier.samples, label: 'content tier' }
-            "
-          ></ng-content>
-        </div>
-      </ng-container>
-      <ng-container *ngIf="tierInfo['metadata-tier']; let metadataTier">
-        <div *ngIf="warningDisplayedTier === DisplayedTier.METADATA">
-          <a (click)="closeWarningView()" class="warning-animated warning-view-title"
-            >Metadata Tier Zero ({{ metadataTier.total }})</a
-          >
-          <ng-content
-            *ngTemplateOutlet="
-              warningViewHeader;
-              context: { list: metadataTier.samples, label: 'metadata tier' }
-            "
-          ></ng-content>
-        </div>
-      </ng-container>
-      <ul *ngIf="warningDisplayedTier === DisplayedTier.CONTENT" class="warning-view-list">
-        <li *ngFor="let recId of tierInfo['content-tier'].samples">
-          <a
-            class="view-record-report"
-            (click)="reportLinkClicked(recId, false)"
-            title="view record report"
-            >{{ recId }}</a
-          >
-        </li>
-      </ul>
-      <ul *ngIf="warningDisplayedTier === DisplayedTier.METADATA" class="warning-view-list">
-        <li *ngFor="let recId of tierInfo['metadata-tier'].samples">
-          <a
-            class="view-record-report"
-            (click)="reportLinkClicked(recId, true)"
-            title="view record report"
-            >{{ recId }}</a
-          >
-        </li>
-      </ul>
+      <sb-navigation-orbs
+        libClickAware
+        [ignoreClasses]="['link-internal', 'nav-orb', 'warning-view-content']"
+        [clickAwareIgnoreWhen]="!showing"
+        (clickOutside)="closeWarningView()"
+        ariaLabel="Tier Zero Warnings Navigation"
+        class="sb-navigation-orbs vertical zero-warning"
+        [count]="getOrbConfigCount()"
+        [fnClassMapInner]="getOrbConfigInner.bind(this)"
+        [fnClassMapOuter]="getOrbConfigOuter.bind(this)"
+        [tooltips]="[
+          'content-tier-zero records found (click to see samples)',
+          'metadata-tier-zero records found (click to see samples)'
+        ]"
+        (clickEvent)="setWarningView($event)"
+      ></sb-navigation-orbs>
+      <div
+        class="warning-view-content"
+        libClickAware
+        [ignoreClasses]="['link-internal', 'nav-orb', 'sb-navigation-orbs']"
+        [clickAwareIgnoreWhen]="!showing"
+        (clickOutside)="closeWarningView()"
+      >
+        <ng-template #warningViewHeader let-list="list" let-label="label">
+          <span class="warning-view-header">
+            This dataset contains
+            {{
+              list.length
```

Comment:
Shouldn't this be `contentTier.total` or `metadataTier.total`? 
Otherwise this will *not* match the actual number of records with tier 0